backend/models/user.js

```diff
@@ -21,5 +22,19 @@ const userSchema = new Schema(
   { timestamps: true }
 );
 
+// static signup method
+userSchema.statics.signup = async function (email, username, password) {
+  const exists = await this.findOne({ $or: [{ email }, { username }] });
+  if (exists) {
+    throw Error("User already exists");
+  }
+
+  const SALT_ROUND = 10;
```

Comment:
Mmm that's weird, I think this is not the last version of the atxr/iscsc.fr:auth branch. I corrected this one in ebe8829e3c8277ddd7751dbff6629033fc5592ca where I put SALT_ROUND in the .env file.
About the security, 10 is the default number of rounds proposed by bcrypt. If you want to have a check read [this](https://stackoverflow.com/questions/46693430/what-are-salt-rounds-and-how-are-salts-stored-in-bcrypt)

Reply:
You're right that's because I reviewed files commit-by-commit thus my comment is about an old commit that has been modified since then.
Tks for the links it helped a lot!